nemo/collections/asr/parts/preprocessing/segment.py

```diff
@@ -358,9 +358,15 @@ def from_file_list(
             else:
                 # Check the dimensions match
                 if len(a_samples) != len(samples):
-                    raise RuntimeError(
-                        f'Loaded samples need to have identical length: {a_samples.shape} != {samples.shape}'
-                    )
+
+                    logging.warning(f"Loaded samples for {a_file}, offset {offset}, duration {duration} have different length from previously loaded ones: {a_samples.shape} != {samples.shape}")
+                    logging.warning("Using zero padding !")
+                    maxlen = max(len(a_samples), len(samples))
+                    a_samples = np.pad(a_samples, ((0, maxlen - len(a_samples)), (0, 0)))
```

Comment:
I have to do some padding here because of the fact that some files in CHiME-6 are longer than others.

Reply:
@tango4j padding is here